packages/vx-demo/src/sandboxes/vx-area/Example.tsx

```diff
@@ -109,15 +109,15 @@ export default withTooltip<AreaProps, TooltipData>(
           />
           <LinearGradient id="area-background-gradient" from={background} to={background2} />
           <LinearGradient id="area-gradient" from={accentColor} to={accentColor} toOpacity={0.1} />
-          <GridRows<number>
```

Comment:
The generic type can be inferred from the input scale and this is no longer needed.